README.md

```diff
@@ -107,6 +107,15 @@ The successful response contains data in `response.data`. The following example
 response.data.create_invoice.first.id
 ```
 
+### Schema Caching and Dumping
+
+To reduce requests to graphql API you can cache schema:
```

Comment:
I think the word cache is misleading here. Cache implies temporary. I would say "you can save to and load schema from disk".